src/messages/index.js

```diff
@@ -1,25 +1,25 @@
 import "./messages.css";
 
-const Messages = () => (
-  <div class="messages">
-    <div id="streak-message" class="show">
-      Current Streak: <span id="streak">@Model.Streak</span>
-    </div>
-    <div id="attempts-message">
-      Attempts Remaining Before Streak Lost: <span id="attempts-left">3</span>
-    </div>
-    <div id="incorrect-display">
-      <div class="incorrect-message">
-        Incorrect!
-        <span id="incorrect-message-reset">Streak Reset.</span>
-      </div>
-      <div class="incorrect-answer">
-        <a href="#answer" id="incorrect-answer-link">
-          Show Answer (Lose Streak)
-        </a>
-      </div>
-    </div>
-  </div>
-);
+const Messages = (props) => {
```

Comment:
Whenever possible, I prefer to use [Destructuring Syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring) instead of `prop`, since it makes it more obvious which properties the component expects

Reply:
const Messages = ({ checkWin }) => {
Like this right?
That way you can see exactly what i'm passing in

Reply:
Yeah that's right 👍